intellij-setup.md

```diff
@@ -13,18 +13,18 @@ This page describes how to set up an IntelliJ IDE to develop Flutter apps.
 ## Flutter IntelliJ IDE plugins
 
 You can write Flutter apps in a text editor, but if you choose to work in an IDE we recommend 
-IntelliJ for a [rich IDE experience](/intellij-ide/). Our Flutter and Dart plug-ins support 
+IntelliJ for a rich IDE experience. Our Flutter and Dart plug-ins support
 editing, running, and debugging Flutter apps.
 
 ### Installing IntelliJ
 
-You can use the IntelliJ plug-ins with one of the following JetBrains IDEs:
+You can use the IntelliJ plugin with the following JetBrains IDEs:
 
-* [IntelliJ IDEA](https://www.jetbrains.com/idea/download/), Community (free) edition, version 2017.1 or later.
-* [IntelliJ IDEA](https://www.jetbrains.com/idea/download/), Ultimate edition, version 2017.1 or later.
-* [IntelliJ WebStorm](https://www.jetbrains.com/webstorm/download), version 2017.1 or later.
+* [IntelliJ CE](https://www.jetbrains.com/idea/download/) Community Edition (CE), version 2017.1 or later.
```

Comment:
Don't we require 2017.2 by now?

Reply:
We support 2017.1 for internal IntelliJ; Android Studio 3.0 is also based on it. We could update to say that more clearly (2017.1 is the min, 2017.2 works and is recommended, 2017.3 coming shortly, ...)

Reply:
I think the current is fine then